m4_sessions/1806-inning/group9.md

```diff
@@ -1,34 +1,55 @@
 ## Group Members:
+Evan Wheeler
+Andrew Piermarini
+Krista Handel
 
 
+## Name of Session: Privacy and Security Awareness in the Digital World
 
-## Name of Session: 
-
-## Session Description 
+## Session Description
 
 (Why should students come to this session): Are you interested in biological enhancement? Drop in to learn about biohacking.
 
-## Session Outcomes 
-
-(What will students learn/leave with?): 
+## Session Outcomes
 
-## Session Outline 
+(What will students learn/leave with?):
 
-(Please include who is facilitating and how much time you are going to spend in each section):
+## Session Outline
 
++ Intros - 15 minutes
++ Topic 1 - Your privacy online - 15 minutes  
```

Comment:
I like this..well thought out.  Please think about creating slides for the presentation or doing a live demo to show what companies do with the data.